### PR TITLE
Feat/favicon missing

### LINKED ISSUE #32: [FEAT]: Favicon missing
The [live](https://dsr-isrowebapp.netlify.app/) link of this repo lacks favicon, kindly implement ISRO icon as the favicon !

### PR DESCRIPTION
**Pull request by pankaj-2503**
### What this PR  introduces 
      In this PR  features is added that is the ISRO  favicon  for  this  project .

### What is the need of this PR
     Favicons can improve the user experience on a website by:
        -Enhancing the usability of the web interface.
        -Supporting brand awareness and recognizability.

Fixes #32

### CODE REVIEW

**Comment by dakshsinghrathore** [comment]
As you can see mozilla logo in the webpage, similarly we are looking for our website ISRO-web!
image

**Review by dakshsinghrathore — CHANGES_REQUESTED** [review]
After reviewing the deployed view, I've noticed that there is no favicon visible.

**Comment by pankaj-2503** [comment]
I have made changes and pushed a new PR you can review it and merge
afterwards

**Review by dakshsinghrathore — COMMENTED** [review]
Attach the screenshot of the deployed website, with favicon showing as it should be!

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/style.css" />
```
Comment (pankaj-2503) [inline index.html:6]: It is showing in website why you are not able to see it i don't understand it

**Comment by pankaj-2503** [comment]
See this is visible here.
Isro-favicon-issue

**Review by dakshsinghrathore — APPROVED** [review]
lgtm !!

**Pull request merged**